src/test/java/org/xmlcml/cml/element/main/CMLAbundanceTest.java

```diff
@@ -0,0 +1,408 @@
+/**
+ *    Copyright 2011 Peter Murray-Rust et. al.
```

Comment:
This should state your own name, ie. of the person who wrote the code. Also, the year should be the year when you wrote the code.

Reply:
Fixed